apps/api/e2e/utils/test-helpers.ts

```diff
@@ -370,3 +371,67 @@ export async function createSiweAuthenticatedClient({
     loginData: loginResponse,
   };
 }
+
+export async function generateTestCompetitions(adminApiKey: string) {
```

Comment:
This is helper that creates 3 users, 3 agents (one for each), and 20 competitions.  agent 1 is participating in 15 comps, agent 2 is in 13, and agent 3 is in 11.  All comps are pending unless the test starts one.

Reply:
not essential, but maybe a comment about this behavior or configurable params would help make sure that it's clear to the dev about what to expect here.